test/ok_test.exs

```diff
@@ -71,7 +71,16 @@ defmodule OKTest do
     assert result == {:ok, 2}
   end
 
-  test "primitives as final operation" do
+  test "primitives as final operation - ok literal" do
```

Comment:
:+1: Still going to organize doctests vs unit tests and whatnot I think. It's going to be _superb_ documentation! :smile: